html5lib/_utils.py

```diff
@@ -27,7 +27,7 @@
         # We need this with u"" because of http://bugs.jython.org/issue2039
         _x = eval('u"\\uD800"')  # pylint:disable=eval-used
         assert isinstance(_x, text_type)
-except:  # pylint:disable=bare-except
+except Exception:  # pylint:disable=bare-except
```

Comment:
Comment is now wrong (though maybe if we  #292 not worth keeping around? but I guess we will?).

Reply:
Thanks for catching. Fixed in the latest revision.